frontend/src/page/Contribute.js

```diff
@@ -1,30 +1,186 @@
-import React from 'react'
+import React, { useState, useEffect } from "react";
+import { useNavigate } from "react-router-dom";
+import ScaleLoader from "react-spinners/ScaleLoader";
 
 function Contribute() {
+  const navigate = useNavigate();
+  const [loading, setLoading] = useState(true);
+  const [error, setError] = useState("");
+  const [message, setMessage] = useState("");
+  const [SongName, setSongName] = useState("");
+  const [Artist, setArtist] = useState("");
+  const [lyrics, setlyrics] = useState("");
+  const [about, setAbout] = useState("");
+  const [url, setUrl] = useState("");
+
+  setTimeout(() => {
```

Comment:
It's a bad practice to make user wait even the page is fully loaded to just show the loader. Pls remove the `setTimeout`